src/quartz_mailer.cr

```diff
@@ -1,73 +1,27 @@
-require "smtp"
-require "yaml"
-require "kilt"
+require "email"
 
 class Quartz::Mailer
-  property client : SMTP::Client
-  property message : SMTP::Message
+  VERSION = "0.4.0"
 
-  def initialize
-    if url = ENV["SMTP_URL"]? || smtp_url
-      if url.includes? ":"
-        host, port = url.split(":")
-      else
-        host, port = url, "25"
-      end
+  def self.deliver(message : Message)
+    config = Quartz.config
 
-      @client = SMTP::Client.new(host, port.to_i)
-      @message = SMTP::Message.new
+    if config.smtp_enabled
+      EMail::Sender.new(
+        config.smtp_address, config.smtp_port,
+        use_tls: config.use_ssl
+      ).start do
```

Comment:
It seems the `config.logger` isn't used for logging in email delivering.
In this case, I think the default logger of `EMail::Client` will be used.
Is this OK?

Reply:
@arcage Good catch. I haven't yet updated the code to use the logger, but I will. :)